lib/internal/fs/recursive_watch.js

```diff
@@ -155,6 +155,7 @@ class FSWatcher extends EventEmitter {
           }
 
           if (file.isFile()) {
+            this.#files.set(f, file);
             this.#watchFile(f);
           } else {
             this.#files.set(f, file);
```

Comment:
Since adding to `#files` becomes unconditional, it can be moved outside of `if` block and we can have one less nesting level.